ci/script.sh

```diff
@@ -25,7 +25,7 @@ case $TARGET in
     ;;
 
   tests)
-    cargo test --workspace --release
+    RUST_MIN_STACK=8388608 cargo test --workspace --release --features runtime-benchmarks
```

Comment:
secondary threads on linux normally start with a slightly smaller stack than the main thread. This was necessary on my laptop to get benchmark tests to run without forcing tests into single-threaded mode.